### DISCUSSION TITLE
How to delete text?

### DISCUSSION THREAD

**Discussion by Number18-tong** [pymupdf/PyMuPDF#2916] [Q&A] [answered]
Can not find the way to delete text in PyMuPDF, how to delete the text by the text content and boundbox in a page? if not have the function right now, will release in the future?

**Comment by JorjMcKie** [comment, marked as answer]
This is already implemented and has been available for a long time now!

Please look at the documentation for ["redaction annotations"](https://pymupdf.readthedocs.io/en/latest/page.html#Page.add_redact_annot), which do exactly that.

I am going to move your post to "Discussions" in case you have questions going forward.

**Reply by Number18-tong** [reply]
Thanks a lot! It's work now, wish you a good day!
